state/watcher.go

```diff
@@ -586,7 +586,10 @@ func (w *modelMachineStartTimeWatcher) loop() error {
 				continue
 			}
 			for _, docID := range changes {
-				unprocessedDocs.Add(w.backend.docID(docID))
+				// filter out doc IDs that correspond to containers
+				if id := w.backend.docID(docID); !strings.ContainsRune(id, '/') {
```

Comment:
Why convert to a <model-uuid>#id just to check if there is a '/' in the id? Probably should do the check first:

if !strings.ContainsRune(doc_id, '/') {
  unprocessedDocs.Add(w.backend.docID(docID))
}

Is it just an ID vs a string type that you're worried about?